Sourcery/Parsing/FileParser.swift

```diff
@@ -144,16 +144,16 @@ final class FileParser {
             case .enumelement:
                 return parseEnumCase(source)
             case .varInstance:
-                return parseVariable(source, containedInProtocol: containingIn is SourceryProtocol)
+                return parseVariable(source, containedIn: containingIn as? Type)
```

Comment:
In your example its fine, just `containedIn: containingIn` is hard to understand. if not `definedIn` then maybe `containingDeclaration` will be more clear?